docs/editor/debugging.md

```diff
@@ -319,6 +319,27 @@ Source maps can be generated with two kinds of inlining:
 
 VS Code supports both the **inlined source maps** and the **inlined source**.
 
+#### JavaScript Source Map Tips
+

```

Comment:
I think that a really simple example of sourcemap debugging isn't that helpful because it's usually easy to set up. Problems come in with more complicated setups, and those are all different.